QUEST_LV_0100.tsv

```diff
@@ -12000,85 +12000,85 @@ QUEST_LV_0100_20151224_011996
 QUEST_LV_0100_20151224_012031	Deliver this letter and Ignas's plot to Irmantas.{nl}{nl}
 QUEST_LV_0100_20151224_012032	I will burden Alina's share of upholding Lydia Schaffen's will as well.{nl}I will find a way to survive and reach Orsha no matter what!{nl}
-QUEST_LV_0100_20151224_012033	I.. wish to repay you for protecting us to the end.{nl}I am sure that Alina and Lydia Schaffen will want that as well.
-QUEST_LV_0100_20151224_012034	This should be enough.. {nl}Now I will take back my shattered soul!
+QUEST_LV_0100_20151224_012033	I... wish to repay you for protecting us to the end.{nl}I am sure that Alina and Lydia Schaffen will want that as well.
+QUEST_LV_0100_20151224_012034	This should be enough...{nl}Now I will take back my shattered soul!
 QUEST_LV_0100_20151224_012035	Agatas honored his side of the contract.{nl}Now I will leave him and start on a journey of revenge.{nl}
 QUEST_LV_0100_20151224_012036	Placed the spirit separation potion in Agatas's pocket.
 QUEST_LV_0100_20151224_012037	I don't really know where he has gone either.{nl}I think Brophen followed the village elder to the Malkos Felled Area, maybe he might know?{nl}
 QUEST_LV_0100_20151224_012038	Anyways, I want to thank you in the meantime.{nl}I hope that the grace of the goddesses will bless your travels.
 QUEST_LV_0100_20151224_012039	Thank you so much for protecting the townfolk on the journey here.{nl}They are all grateful for your kindness.
-QUEST_LV_0100_20151224_012040	How many Migration Orders did Orsha issue anyways..{nl}I don't know how much longer we will have to just wait here.
-QUEST_LV_0100_20151224_012041	I don't know how many times you've protected our townsfolk on the way here..{nl}Yet we receive your help again. Thank you so much.{nl}
+QUEST_LV_0100_20151224_012040	How many migration orders did Orsha issue anyways...{nl}I don't know how much longer we will have to just wait here.
+QUEST_LV_0100_20151224_012041	I don't know how many times you've protected our townsfolk on the way here...{nl}Yet we receive your help again. Thank you so much.{nl}
 QUEST_LV_0100_20151224_012042	By the way, how are the nightmares now?{nl}How about consulting the bishop in Orsha when you get there?
-QUEST_LV_0100_20151224_012043	The lord of Orsha should have issued Migration Orders sooner.{nl}There were too many casualties, too many..{nl}
-QUEST_LV_0100_20151224_012044	It's in times like this when you need something to rely upon..{nl}Where have the goddesses gone to..
-QUEST_LV_0100_20151224_012045	They said that the lord will send soldiers and officials to help if we managed to get here..{nl}I don't see any sign of them.{nl}
+QUEST_LV_0100_20151224_012043	The lord of Orsha should has issued the migration orders sooner.{nl}There were too many casualties. Too many...{nl}
+QUEST_LV_0100_20151224_012044	It's in times like this when you need something to rely upon...{nl}Where have the goddesses gone to...
```

Comment:
It is in times like this when you need someone to have faith in...? ('have faith in' is to 'rely' on something thru faith, imo)